tools/run_integration_tests.sh

```diff
@@ -56,7 +56,12 @@ check_required_params
 
 # When tests run, they run in the root of the module directory. Anything
 # relative to our current directory won't work properly
-GCS_TEST_PRIVATE_KEYFILE=$(readlink -f "${GCS_TEST_PRIVATE_KEYFILE}")
+if [[ $(uname -s) == Darwin ]]; then
+  # On MacOS `readlink` does not support `-e` parameter
+  GCS_TEST_PRIVATE_KEYFILE=$(readlink -n "${GCS_TEST_PRIVATE_KEYFILE}")
```

Comment:
What would the behaviour here if an absolute credential file path is passed?

Reply:
Fixed it and added assert for keyfile existence after absolute path and symlink resolution.

Reply:
Great Thanks :)